amgut/db/patches/0036.sql

```diff
@@ -0,0 +1,389 @@
+-- May 31, 2016
+-- Add surfer and fermeted foods questionnaires
+
+----------------------------------------------------------
+-- survey_group
+----------------------------------------------------------
+INSERT INTO survey_group (group_order, american, british) VALUES (-3, 'Fermented Foods', 'Fermented Foods');
+INSERT INTO survey_group (group_order, american, british) VALUES (-4, 'Surfers', 'Surfers');
+
+----------------------------------------------------------
+-- surveys
+----------------------------------------------------------
+INSERT INTO surveys (survey_id, survey_group) VALUES (3, -3);
+INSERT INTO surveys (survey_id, survey_group) VALUES (4, -4);
+
+----------------------------------------------------------
+-- survey_question
+----------------------------------------------------------
+-- Fermented
+INSERT INTO survey_question (survey_question_id, question_shortname, american, british) VALUES
+(164, 'FERMENTED_FREQUENCY', 'How often do you consume one or more servings of fermented vegetables or plant products a day in an average week? (1 serving = ½ cup sauerkraut, kimchi or fermented vegetable or 1 cup of kombucha)', 'How often do you consume one or more servings of fermented vegetables or plant products a day in an average week? (1 serving = ½ cup sauerkraut, kimchi or fermented vegetable or 1 cup of kombucha)'),
```

Comment:
Has this code being exercised through the interface? I'm concerned about the `½` characters present on this text. Given the history of non-ascii characters errors I prefer to double-check. I'm sure that this is a result of just copy/pasting the text.

I will feel more comfortable using `1/2` rather than those characters. @antgonza @wasade what do you think?